lib/runtime.js

```diff
@@ -1,5 +1,6 @@
 var Entry = require("./entry.js").Entry;
 var utils = require("./utils.js");
+var dynRequire = module.require || __non_webpack_require__;
 
 exports.enable = function (Module) {
```

Comment:
Cool, I'll update. Is there a unit test to check that as currently they're passing.